.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2
+jobs:
+  relay:
+    working_directory: ~/artsy/reaction
+    docker:
+    - image: circleci/node:8
+    steps: 
+      - checkout
+      - restore_cache:
+          keys:
+          - yarn-deps-{{ checksum "yarn.lock" }}
+          - yarn-deps-
+      - run: yarn --check-files
```

Comment:
Because we're caching `node_modules` and there's a potential that our dependencies won't be present or up-to-date, we call yarn with the `--check-files` flag. It does what it says... looks through the node_modules folder to see if anything is missing. If nothing is missing, it quits as per usual, else it installs the missing stuff.